src/Combinatorics/SimplicialComplexes.jl

```diff
@@ -0,0 +1,353 @@
+    R, () = PolynomialRing(ZZ, n)
+    return stanley_reisner_ring(R, K)
+end
+
+@doc Markdown.doc"""
+    stanley_reisner_ring(R::FmpzMPolyRing, K::SimplicialComplex)
+
+Stanley-Reisner ring of the abstract simplicial complex `K`, as a quotient of a given ring `R`.
+
+# Example
+```jldoctest
+julia>  R, () = ZZ["a","b","c","d","e","f"];
+
+julia> stanley_reisner_ring(R, real_projective_plane())
+(Quotient of Multivariate Polynomial Ring in 6 variables a, b, c, d, ..., f over Integer Ring by ideal(a*b*c, a*b*d, a*e*f, b*e*f, a*c*f, a*d*e, c*d*e, c*d*f, b*c*e, b*d*f), Map from
+Multivariate Polynomial Ring in 6 variables a, b, c, d, ..., f over Integer Ring to Quotient of Multivariate Polynomial Ring in 6 variables a, b, c, d, ..., f over Integer Ring by ideal(a*b*c, a*b*d, a*e*f, b*e*f, a*c*f, a*d*e, c*d*e, c*d*f, b*c*e, b*d*f) defined by a julia-function with inverse)
+```
+"""
+stanley_reisner_ring(R::FmpzMPolyRing, K::SimplicialComplex) = quo(R, stanley_reisner_ideal(R, K))
+
+################################################################################
+##  Standard examples
+################################################################################
+
+"""
+    torus()
+
+Császár's 7-vertex triangulation of the torus (surface).
+"""
+torus() = SimplicialComplex(Polymake.topaz.torus())
+
+"""
+    klein_bottle()
+
+9-vertex triangulation of the Klein bottle.
+"""
+klein_bottle() = SimplicialComplex(Polymake.topaz.klein_bottle())
+
+"""
+    real_projective_plane()
+
+6-vertex triangulation of the real projective plane.
+"""
+real_projective_plane() = SimplicialComplex(Polymake.topaz.real_projective_plane())
+
+"""
+    complex_projective_plane()
+
+9-vertex triangulation of the complex projective plane.
+"""
+complex_projective_plane() = SimplicialComplex(Polymake.topaz.complex_projective_plane())
+
+###############################################################################
+### Display
+###############################################################################
+
+function Base.show(io::IO, K::SimplicialComplex)
+    d = dim(K)
+    n = nvertices(K)
+    print(io, "Abstract simplicial complex of dimension $(d) on $(n) vertices")
+end
+
+###############################################################################
+### Serialization
+###############################################################################
+
+"""
+    save_simplicialcomplex(K::SimplicialComplex, filename::String)
+
+Save a SimplicialComplex to a file in JSON format.
+"""
+function save_simplicialcomplex(K::SimplicialComplex, filename::String)
+    bigobject = pm_object(K)
+    Polymake.save_bigobject(bigobject, filename)
+end
+
+"""
+    load_simplicialcomplex(filename::String)
+
+Load a SimplicialComplex stored in JSON format, given the filename as input.
+"""
+function load_simplicialcomplex(filename::String)
+   bigobject = Polymake.load_bigobject(filename)
+   typename = Polymake.type_name(bigobject)
+   if typename[1:17] != "SimplicialComplex"
```

Comment:
So why is this restricted to 17 characters on the left side? So we also accept objects with typename `SimplicialComplexffhjficknndkxif` ?

Reply:
That does not occur in polymake.  So that's not an issue here.

@lkastner : please refactor load/save mechanism after merge

Reply:
Let me phrase it differently: why doesn't this code just say:
```suggestion
   if typename != "SimplicialComplex"
```